dot/parachain/availability-store/availability_store.go

```diff
@@ -83,23 +205,33 @@ func (as *AvailabilityStore) loadMetaData(candidate common.Hash) (*CandidateMeta
 }
 
 // storeMetaData stores metadata in the availability store
-func (as *AvailabilityStore) storeMetaData(candidate common.Hash, meta CandidateMeta) error {
+func (as *AvailabilityStore) writeMeta(batch *AvailabilityStoreBatch, candidate parachaintypes.CandidateHash,
+	meta CandidateMeta) error {
```

Comment:
nit: I would use `writeMetadata` instead of `writeMeta`. Is parity calling it meta as well?

update comment!

Reply:
Yes, Parity is calling it writeMeta, which is why I updated it to conform to their naming.

Reply:
I see. I think I still have a mental block for it, haha